test/base/BaseModel.spec.js

```diff
@@ -4,10 +4,39 @@ import _isString from 'lodash/isString';
 import BaseModel from '../../src/assets/scripts/client/base/BaseModel';
 import ExtendedBaseModelFixture from './_fixtures/ExtendedBaseModelFixture';
 
+ava('sets the #_id when passed valid parameters', (t) => {
```

Comment:
this test can be improved a bit.
1) if you're going to do multiple assertions inside of a test you should use a single var to hold the results:
```javascript
let model = new BaseModel();
// ...
model = new BaseModel('string');
```

2) use a more realistic example for the passed model name:
```javascript
// instead of
modelBool = new BaseModel('string');

// do
modelBool = new BaseModel('modelName');
```  This adds a little more context to the test and informs a future developer about the intent.